source/YearlyOverview/YearlyOverview.html

```diff
@@ -10,10 +10,16 @@
     <body>
         <div id="header">
             <h1 id="index">
-                <a href="../Index/Index.html" id="back"> Index</a>
+                <a
+                    href="http://127.0.0.1:5500/source/Index/Index.html"
```

Comment:
just like daily and monthly overview, I think it would be good to keep the ".." notation